datadog/tests/cassettes/TestAccDatadogLogsPipeline_basic.yaml

```diff
@@ -13,7 +13,7 @@ interactions:
         remote_addr: ""
```

Comment:
I'm not sure why, but the test fails if I don't update these mock responses. Since the test passes when using the "real" API, I suppose we don't need to worry about it too much.